examples/A2GrantAppDashboardAccess.md

```diff
@@ -0,0 +1,45 @@
+# HowTo - Grant Access to the Applications Dashboard in Automate 2
+
+These steps will grant blanket access to the Applications view in Automate 2 for all users.
+
+## Before You Start
+
+### Assumptions
+
+* Chef Automate 2 Installed
+* curl installed and running locally.
+
+### Tested Versions
+
+* Chef Automate 2 | [`20200603114954`]
+
+## Grant Access to the Applications Dashboard in Automate 2
+
+1. Create JSON policy definition in a text file (in this example, we'll call it `policy`): 
+```
+{
+  "name": "Applications Viewer",
+  "id": "applications-viewer-access",
+  "members": [
+    "*"
+  ],
+  "statements": [
+    {
+      "effect": "ALLOW",
+      "actions": [
+        "applications:*"
+      ],
+      "projects": [
+        "*"
+      ]
+    }
+  ]
+}
+```
+2. POST the policy. I used an admin token, but any token that can manage IAM policies should work:
+```
+export TOKEN=`chef-automate iam token create admoon --admin`
```

Comment:
@scottvidmar Looks like a typo in the command `admoon`?

Reply:
Not a typo, but I'm changing it to be more clear that this is the token's name being created.